src/components/Dialog/Dialog.stories.mdx

```diff
@@ -1,14 +1,14 @@
-import { Canvas, Meta, Story, ArgsTable } from "@storybook/addon-docs";
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
 
-import Dialog from "./Dialog";
-import { dialog, argTypes } from "./Dialog.stories.tsx";
+import Dialog from './Dialog';
+import { dialog, argTypes } from './Dialog.stories.tsx';
 
 <Meta
-  title="Molecules/Dialog"
+  title='Molecules/Dialog'
```

Comment:
wait what setting is this 🤔 I don't think we want single quotes for string props (or rather, that's not how we usually do it in `izakaya`)